src/foam/dao/EasyDAO.js

```diff
@@ -211,6 +201,11 @@ if ( getOrder() != null &&
   }
 }
 
+if ( getPermissioned() &&
```

Comment:
It defaults to true.  If we don't provide for turning it off then than one DAO setup that needs it can't use EasyDAO and will cause Cluster setup issues.